src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  isError: string;
+  setIsError: () => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({ isError, setIsError }) => {
+  useEffect(() => {
+    setTimeout(() => {
+      setIsError();
+    }, 3000);
```

Comment:
You should clear all timeouts on the component unmount https://bobbyhadz.com/blog/react-cleartimeout